virttest/utils_v2v.py

```diff
@@ -742,31 +742,45 @@ def import_vm_to_ovirt(params, address_cache, timeout=600):
     return True
 
 
-def check_log(log, content_list, expect=True):
+def check_log(params, log):
     """
-    Check if Error Message exists in v2v log
+    Check if error/warning meets expectation in v2v log
```

Comment:
I think that better way is o give a pattern list  for your message,  if message match pattern return True, else return False.